homeassistant/components/whirlpool/__init__.py

```diff
@@ -27,8 +26,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     session = async_get_clientsession(hass)
     region = CONF_REGIONS_MAP[entry.data.get(CONF_REGION, "EU")]
-    brand = get_brand_for_region(region)
+    brand = CONF_BRANDS_MAP[entry.data.get(CONF_BRAND, "Whirlpool")]
```

Comment:
This appears to work even though US originally defaulted to Maytag.  
I'm not sure that it will work for everyone, but in the "worst case" the user would need to delete and re-install the integration.